eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -18,7 +20,7 @@ steps:
         --kusto-database-var KUSTO_DB `
         --kusto-table-var KUSTO_TABLE `
         --identity "$(Build.QueuedBy)" `
-        --targetvar "${{ parameters.TargetVariable }}"
+        --targetvar "${{ coalesce(parameters.TargetVariable, parameters.TargetUserVariable) }}"
```

Comment:
If these are both not specified, meaning they are optional parameters, the coalesce statement will fail because it requires a non-empty value for assignment (I wish it behaved differently).